src/index.js

```diff
@@ -108,19 +108,23 @@ async function main() {
 
   console.log('xcmpCall: ', xcmpCall);
 
-  console.log(`\n2. Estimating XCM fees ...`);
+  console.log(`\n2. Query automationTime fee details `);
+  const { executionFee, xcmpFee } = await turingHelper.api.rpc.automationTime.queryFeeDetails(xcmpCall);
+  console.log(`automationFeeDetails: `, { executionFee: executionFee.toHuman(), xcmpFee: xcmpFee.toHuman() });
+
+  console.log(`\n3. Estimating XCM fees ...`);
   const xcmFrees = await turingHelper.getXcmFees(turingAddress, xcmpCall);
   console.log("xcmFrees:", xcmFrees.toHuman());
```

Comment:
the result xcmpFee from `queryFeeDetails` is supposed to replace this `getXcmFees`, correct?

@nvengal

Reply:
Yes, that's correct. `getXcmFee` is the old version.

Reply:
okay, Charles let’s remove the old `xcmFrees` variable and use the one returned from queryFeedDetails.

Reply:
@imstar15 this line still needs update.